### PR TITLE
Feat/information box

### PR DESCRIPTION
**Pull request by Adityapnn811**
Added information box on hover. The behavior and the information shown is the same as Satrio's classviz

image

### CODE REVIEW

File: svelte/src/example-raw-data/simple-data.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export const simpleData: RawInputType = {
 					properties: {
 						simpleName: 'main',
 						kind: 'class',
+						description: 'main class where the program starts',
```
Comment (christojeffrey) [inline svelte/src/example-raw-data/simple-data.ts:12]: the attributes name is based on the example data given by satrio right?

**Review by christojeffrey — COMMENTED** [review]
functionally looks good

File: svelte/src/example-raw-data/simple-data.ts (hunk 1 above)
Comment (Adityapnn811) [reply svelte/src/example-raw-data/simple-data.ts:12]: It just accepts and converts the data from the raw data. I don't think it cares about the attribute names, as it just save whatever given to it. But yes so far it should be based on what Satrio has made.

Comment (christojeffrey) [reply svelte/src/example-raw-data/simple-data.ts:12]: well looks good to me!

**Review by christojeffrey — APPROVED** [review]

**Pull request merged**